scan/serverapi.go

```diff
@@ -700,6 +700,7 @@ func ViaHTTP(header http.Header, body string) (models.ScanResult, error) {
 		Packages:    installedPackages,
 		SrcPackages: srcPackages,
 		ScannedCves: models.VulnInfos{},
+		ReportedAt: time.Now(),
```

Comment:
I think it is better to set reportedAt here to cover `application / json` pattern. Not here.
https://github.com/future-architect/vuls/blob/fe3f1b99245266e848f7b8f240f1f81ae3ff04df/server/server.go#L61